src/app/app.component.ts

```diff
@@ -2,9 +2,6 @@ import { Component } from '@angular/core';
 
 @Component({
   selector: 'app-root',
-  templateUrl: './app.component.html',
-  styleUrls: ['./app.component.css']
+  template: '<router-outlet></router-outlet>',
```

Comment:
As per my last comment, you shouldn't use a router here, you should directly use ``app-country``